src/Tasks/Common/Resources/xlf/Strings.ko.xlf~RF26a08dc.TMP

```diff
@@ -0,0 +1,815 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this file should be removed.

Reply:
Damn it, I thought I had excluded it, thanks for spotting, hopefully should be gone now.